Shared/Extensions/JellyfinAPI/BaseItemDto+Images.swift

```diff
@@ -80,11 +80,13 @@ extension BaseItemDto {
         maxWidth: Int?,
         maxHeight: Int?,
         itemID: String
-    ) -> URL {
+    ) -> URL? {
+        guard let imageTags = imageTags else { return nil }
+        
         // TODO: See if the scaling is actually right so that it isn't so big
         let scaleWidth = maxWidth == nil ? nil : UIScreen.main.scale(maxWidth!)
         let scaleHeight = maxHeight == nil ? nil : UIScreen.main.scale(maxHeight!)
-        let tag = imageTags?[type.rawValue]
+        let tag = imageTags[type.rawValue]
```

Comment:
Checking the `imageTags` is the first step, the next is checking if this tag has a value and if not also `return nil`

Reply:
Additionally, we will have to make sure the fields for the tags are requested along with the items.